### ISSUE TITLE
toJsonArray can lead to infinite recursion

### ISSUE THREAD

**Issue by GoogleCodeExporter** [manojjsm/js-test-driver#250]
```
I'm seeing toJsonArray (in runner.js) produce infinite recursion when 
serialising Backbone.js models, and HTML elements.

It'd be great if it had a limit - I tried a simple hack but it looks like 
errors thrown from there don't cause a failed test, but the test suite to never 
complete.
```

Original issue reported on code.google.com by `timruff...@googlemail.com` on 5 Jul 2011 at 10:33

Attachments:
- [infinite recursion backbone.json](https://storage.googleapis.com/google-code-attachments/js-test-driver/issue-250/comment-0/infinite recursion backbone.json)
- [infinite recursion.json](https://storage.googleapis.com/google-code-attachments/js-test-driver/issue-250/comment-0/infinite recursion.json)

**Comment by GoogleCodeExporter** [comment]
```
Sorry - forgot to add, this normally happens when serialising an error.
```

Original comment by `timruff...@googlemail.com` on 5 Jul 2011 at 10:40

**Comment by GoogleCodeExporter** [comment]
```
Should be resolved in 1.3.3a, as of r975.
```

Original comment by `corbinrs...@gmail.com` on 29 Sep 2011 at 3:05

**Comment by GoogleCodeExporter** [comment]
Original comment by `corbinrs...@gmail.com` on 29 Sep 2011 at 3:06
- Changed state: **Fixed**
